polling_stations/apps/data_collection/management/commands/import_tameside.py

```diff
@@ -3,23 +3,89 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E08000008"
-    addresses_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018.tsv"
-    stations_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018.tsv"
-    elections = ["local.2018-05-03"]
-    csv_delimiter = "\t"
+    addresses_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019.CSV"
+    stations_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019.CSV"
+    elections = ["local.2019-05-02"]
 
     def address_record_to_dict(self, record):
         uprn = record.property_urn.strip().lstrip("0")
-
-        if record.addressline6 == "OL5 0EX":
-            return None
+        rec = super().address_record_to_dict(record)
+        postcode = record.addressline6
+        #     if record.addressline6 == "OL5 0EX":
+        #         return None
```

Comment:
if we don't want this, can we just remove it

Reply:
:+1: